README.md

```diff
@@ -86,6 +87,38 @@ Logging and metrics are only active when they are configured (see below). This
 means you can add your own logging or metrics libraries without conflict, but
 will miss out on the free built-in support.
 
+## Asynchronous Dispatch
+
+GitHub imposes timeouts on webhook delivery responses. If an application does
+not respond in time, GitHub closes the connection and marks the delivery as
+failed. `go-githubapp` optionally supports asynchronous dispatch to help solve
+this problem. When enabled, the event dispatch sends a response to GitHub after
+validating the payload and then runs the event handler in a separate goroutine.
+
+To enable, select an appropriate _scheduler_ and configure the event dispatcher
+to use it:
+
+```go
+dispatcher := githubapp.NewEventDispatcher(handlers, secret, githubapp.WithScheduler(
+    githubapp.AsyncScheduler(),
+))
+```
+
+The following schedulers are included in the library:
+
+- `DefaultScheduler` - a synchronous scheduler that runs event handlers in
```

Comment:
I'm using default because these are the functions/types that are used when the caller does not override them via options. In this sense, it matches `DefaultErrorCallback`, `DefaultResponseCallback`, and `NewDefaultEventDispatcher`, which already exist.

I'm open to renaming the scheduler if you have suggestions and then documenting the default-ness in comments. I'm less convinced that renaming the other functions will improve clarity.